steps/appeal-form-download/content.en.json

```diff
@@ -1,8 +1,8 @@
 {
-  "titleHead": "Download appeal form - Appeal a benefit decision - GOV.UK",
+  "titleHead": "Download appeal form",
```

Comment:
why this title head change, any perticuler reason

Reply:
`windowTitleSuffix` has been added to `commonContent.json` and is now being appended to all templates

Reply:
https://github.com/hmcts/sscs-submit-your-appeal/blob/master/commonContent.json